### PR TITLE
Add uptime checks

### PR DESCRIPTION
**Pull request by tiagojsag**
the user purge scheduler should be merged first, and then rebase

### CODE REVIEW

**Comment by agnessa** [comment]
did you say rebase? 😈 😈 😈

**Comment by tiagojsag** [comment]
rebase is the true sign of git proficiency. I already did it, ready for review

File: infrastructure/base/modules/env/main.tf [hunk 1]
```diff
@@ -391,6 +391,21 @@ module "purge_users_cron" {
...
+  name   = "${var.project_name} Frontend"
+  host   = var.domain
+  email  = var.uptime_alert_email
+}
+
+module "api_uptime_check" {
+  source = "../uptime-check"
+  name   = "${var.project_name} API"
+  host   = var.domain
+  path   = "/backend/health_check"
```
Comment (agnessa) [inline infrastructure/base/modules/env/main.tf:405]: I understand we check BE availability by using the health endpoint, but I don't understand how we check FE, i.e. there's no path given in the FE uptime check?

Comment (tiagojsag) [reply infrastructure/base/modules/env/main.tf:405]: correct, there is no path, we check if the homepage is up

Comment (agnessa) [reply infrastructure/base/modules/env/main.tf:405]: 🧙

**Review by agnessa — APPROVED** [review]

**Pull request merged**